msteams-platform/toolkit/teams-toolkit-fundamentals.md

```diff
@@ -53,72 +53,72 @@ Let's take a tour of the topics covered in this document:
 
 ## Accounts
 
-To develop a Teams app, you need at least one Microsoft 365 account with a valid subscription. If you want to host your backend resources on Azure, an Azure account is also needed. Teams Toolkit support integrated experience to sign in, provision and deployment for Azure resources. You can [create a free Azure account](https://azure.microsoft.com/free/) before you start.
+To develop a Teams app you need a Microsoft 365 account with a valid subscription. If you want to host your backend resources on Azure, an Azure account is also needed. Teams Toolkit supports an integrated sign-in experience to provision and deploy Azure resources. If needed, you can [create a free Azure account](https://azure.microsoft.com/free/) before you start.
 
 ## Environment
 
-Teams Toolkit helps you to create and manage multiple environments, provision, and deploy artifacts to the target environment for Teams App.
+Teams Toolkit helps you to create and manage multiple environments, provision resources, and deploy artifacts to the target environment for the Teams app.
 
 ### TeamsFx Collaboration
 
-It allows developers and project owner to invite other collaborators to the TeamsFx project to debug, provision, and deploy the same TeamsFx project.
+It allows developers and project owners to invite other collaborators to the TeamsFx project to debug, provision, and deploy the project.
 
 ## Development
 
-Teams Toolkit helps you to create and customize your Teams app project that makes the Teams app development work simpler.
+Teams Toolkit helps you to create and customize your Teams app project, and makes simplifies Teams app development.
```

Comment:
Teams Toolkit helps you to create and customize your Teams app project, and simplifies Teams app development.

Reply:
good catch :-)

Reply:
Thanks